.github/workflows/test.yml

```diff
@@ -12,35 +12,48 @@ jobs:
         gemfile: [gemfiles/rails_6.1.gemfile]
         orm: [active_record]
         adapter: [sqlite3]
+        asset: [sprockets]
```

Comment:
If we go this way,

```yaml
ruby: [2.6, 2.7, 3.0, jruby]
asset: [sprockets, webpacker]
```

we'll get every combination of 2.6/sprockets, 2.6/webpacker, 2.7/sprockets, ... and so on. That's kind of redundant, since in general Ruby version will not affect how JavaScript assets are executed.

So I didn't add webpacker in the global matrix, and added it individually like this to save CI execution time and computer resource. 

```yaml
        include:
          ...
          - ruby: 3.0
            gemfile: gemfiles/rails_6.1.gemfile
            orm: active_record
            adapter: postgresql
            asset: webpacker
          - ruby: 3.0
            gemfile: gemfiles/rails_7.0.gemfile
            orm: active_record
            adapter: sqlite3
            asset: webpacker
```